Localize/lang/strings.json

```
@@ -7,6 +7,7 @@
   "+JCU+Y": "Select this if you are configuring an on-prem connection",
   "+W2vTp": "Please enter an e-mail address",
   "+W9fkq": "(UTC+08:00) Kuala Lumpur, Singapore",
+  "+ZSBrq": "Context menu for {title} card",
   "+dlOgj": "(UTC-05:00) Haiti",
   "+g+mch": "Returns the body for a part in a multipart output of an action",
   "+iMTt7": "Check to select this connection",
```

- Use "select", not "check". See ["check" - Microsoft Style Guide](https://learn.microsoft.com/style-guide/a-z-word-list-term-collections/c/check-checkbox-check-mark).

```suggestion
  "+iMTt7": "Select the checkbox to choose this connection.",
```